src/app/routes/loadInitialData/getDials.js

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs';
+
+const transformDial = value => {
+  switch (value) {
+    case 'true':
+      return true;
+    case 'false':
+      return false;
+    default:
+      return value;
+  }
+};
+
+const getDials = () => {
+  const dialsPath = process.env.COSMOS_DIALS_PATH;
+
+  if (!dialsPath) {
+    return {};
+  }
+
+  const buffer = fs.readFileSync(dialsPath);
+  const dials = JSON.parse(buffer);
+
+  Object.keys(dials).forEach(dial => {
+    const value = dials[dial];
+    dials[dial] = transformDial(value);
+  });
+
+  return dials;
```

Comment:
I feel like some error handling and logging (via nodeLogger) might be helpful here in debugging down possible issues that might arise here (e.g. file missing/corrupt/empty).